src/Test/MakerTestDetails.php

```diff
@@ -352,4 +352,13 @@ public function getSkipMessage(): ?string
     {
         return $this->skipMessage;
     }
+
+    public function useDoctrineAttributeMapping(): self
+    {
+        return $this->addReplacement(
+        'config/packages/doctrine.yaml',
+                'type: annotation',
+                'type: attribute'
+        );
+    }
```

Comment:
This will likely break not long after merging this PR, as it will allow for this recipe to be merged: https://github.com/symfony/recipes/pull/971/files

I can't immediately think of anything better to do. So, we might just need to wait for that recipe to break our tests and then fix them.

Reply:
I'll keep an eye on the PR and our MB test results once we merge this so we can push a fix for the tests pretty quick.